discourse_rock/patches/rake.patch

```diff
@@ -0,0 +1,87 @@
+--- a/lib/tasks/discourse-charm.rake
++++ b/lib/tasks/discourse-charm.rake
+@@ -0,0 +1,84 @@
++# frozen_string_literal: true
++
++desc "Promote a user to admin"
++task "admin:promote", [:email] => [:environment] do |_, args|
```

Comment:
We are creating code that cannot be tested by the charm unit tests. We are testing by luck in the integration tests. I would rather create tests in Ruby, or at least have one specific integration test in the charm that tests this code on purpose.